dist/stat_data.py

```diff
@@ -210,9 +214,19 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_pages_inuse',
         'pages currently held in the cache', 'no_clear,no_scale'),
     CacheStat('cache_read', 'pages read into cache'),
+    CacheStat('cache_read_lookaside',
+        'pages read into cache requiring lookaside entries'),
```

Comment:
> I'd like to simplify the stats, if possible, so we know what to look for to decide (1) whether updates are being written to the lookaside table, (2) whether reads are having to check it and (3) if it is growing very large.  Maybe just have `cache_read_lookaside` and rename `rec_pages_lookaside` to `cache_write_lookaside`?

I've reduced it to:

```
Connection only:
    cache_lookaside_insert
    cache_lookaside_remove

Connection & data-source:
    cache_read_lookaside
    cache_write_lookaside
    cache_write_restore
```

For (1), cache_write_lookaside, for (2) cache_read_lookaside, and (3) cache_lookaside_insert vs. cache_lookaside_remove.

I've left cache_write_restore even though it's not related to the lookaside table. That's a new statistics that tells us if we're updating/restoring pages because we had uncommitted updates on the page.

Let me know if that's still not right.